components/search_bar/search_bar.tsx

```diff
@@ -115,7 +115,7 @@ const SearchBar: React.FunctionComponent<Props> = (props: Props): JSX.Element =>
                     ref={getSearch}
                     id={props.isSideBarRight ? 'sbrSearchBox' : 'searchBox'}
                     tabIndex='0'
-                    className='search-bar a11y__region'
+                    className='search-bar form-control'
```

Comment:
Removing `a11y__region` has accessibility implications. We use that class to hep facilitate navigation through the app using the keyboard. Is there a specific reason we are removing that class?

Reply:
I thought it gets added automatically and here it was added manually ... I can re-add it if it is not added automatically

Reply:
Yes let's re-add that.

Reply:
Did re-add it. Could you check again and confirm? Thanks! 🙇🏼‍♂️